src/Makefile

```diff
@@ -16,7 +16,7 @@ DETOURS_SOURCE_BROWSING = 0
 
 #######################/#######################################################
 ##
-CFLAGS=/nologo /W4 /WX /we4800 /Zi /MT /Gy /Gm- /Zl /Od
```

Comment:
What happens with old compiler?

Reply:
I tested a warning introduced in VS 2019 on a VS 2017 version of MSVC, and it seemed to just ignore it.